### PR TITLE
upgrade with the latest black version 21.4b0

### PR DESCRIPTION
**Pull request by harshad16**
updates based on the new black update https://github.com/psf/black/releases/tag/21.4b0
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Fixes: https://github.com/tomcatling/black-nb/issues/58
Related-to: https://github.com/aicoe-aiops/ocp-ci-analysis/issues/246

### CODE REVIEW

File: black_nb/cli.py [hunk 1]
```diff
@@ -72,6 +77,23 @@
     ),
     show_default=True,
 )
+@click.option(
+    "--extend-exclude",
+    type=str,
```
Comment (liamcoatman) [inline black_nb/cli.py:82]: Can you add `callback=black.validate_regex,` to `extend-exclude`, `include` and `exclude`? It returns a `Pattern[str]` type which is needed by `black.get_sources`

File: black_nb/cli.py [hunk 2]
```diff
@@ -128,6 +150,8 @@ def cli(
     check: bool,
     include: str,
     exclude: str,
+    extend_exclude: str,
```
Comment (liamcoatman) [inline black_nb/cli.py:153]: After adding the `validate_regex` callback, the type of `include` and exclude should be `Pattern[str]`. The type of `extend_exclude` should be `Optional[Pattern[str]]`. The type of `stdin_filename` should be `Optional[str]`.

**Review by liamcoatman — CHANGES_REQUESTED** [review]
Thanks @harshad16. I've left a couple of comments that should be easy enough to fix.

File: black_nb/cli.py (hunk 1 above)
Comment (harshad16) [reply black_nb/cli.py:84]: Absolutely, I have updated these changes.

File: black_nb/cli.py (hunk 2 above)
Comment (harshad16) [reply black_nb/cli.py]: updated the type as well :+1:

**Review by liamcoatman — APPROVED** [review]

**Comment by liamcoatman** [comment]
Thanks a lot @harshad16 !

**Pull request merged**